.gitignore

```diff
@@ -0,0 +1,15 @@
+.classpath
+.jekyll-cache/
+.project
+_site/
+Gemfile.lock
+
+## VS Code
+
+.settings/
+.vscode/
+
+## Inproper copy when switching branch
```

Comment:
Kindly give a moment to check this on my computer. Just to see the intended effect.